app/main/posts/savedsearches/savedsearches-routes.js

```diff
@@ -0,0 +1,98 @@
+module.exports = [
+    '$stateProvider',
+    function (
+        $stateProvider
+    ) {
+
+    $stateProvider
+    .state(
+        {
+            url: '^/savedsearches/:id',
+            name: 'savedsearchesRedirector',
+            params: {
+                id: null
+            },
+            onEnter: function ($state, savedSearch) {
+                var viewParam = savedSearch.view;
```

Comment:
same as for collections, take the user where they want to go with their default values